.github/workflows/integration-tests.yml

```diff
@@ -104,16 +104,22 @@ jobs:
           echo "node=node_${node/.*.*/}" >> $env:GITHUB_OUTPUT
         shell: bash
         if: '${{ !steps.release-check.outputs.IS_RELEASE }}'
+
+      - name: Sanitize shard for artefact name
+        id: sanitize-shard-name
+        run: echo "shard=$(echo '${{ matrix.shard }}' | tr '/' '-')" >> $GITHUB_OUTPUT
+
       - name: Store npm error artefacts
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         if: always()
         with:
-          name: npm-logs
+          name: npm-logs--${{ matrix.os }}--${{ matrix.node-version }}--${{ steps.sanitize-shard-name.outputs.shard }}
           path: |
             /home/runner/.npm/_logs/**/*
 
       - uses: codecov/codecov-action@v4
         continue-on-error: true
         with:
           flags: ${{ steps.test-coverage-flags.outputs.os }},${{ steps.test-coverage-flags.outputs.node }}
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
This is currently not being picked up, could we make sure it hasn't accidentally been removed?

Reply:
Do we have an account for https://app.codecov.io/gh/netlify/cli? Can we retrieve a token from that account? Otherwise I can downgrade us back to `codecov/codecov-action@v3` (where the token isn't required for the workflow step). There might be a case to be made for removing the codecov step completely as it doesn't seem to be very depended on, though you folks would have more context on that topic.